account_invoice_validate_send_email/views/account_invoice_view.xml

```diff
@@ -9,6 +9,10 @@
                 <field name="invoice_sent" />
                 <field name="web_url" invisible="1" />
             </xpath>
+            <xpath expr="//field[@name='move_id']" position="after">
+                <field name="carrier_info_name" />
+                <field name="carrier_info_url" />
+            </xpath>
```

Comment:
I suppose it's not necessary to show the carrier fields. We currently do not show the tracking refs either.

Reply:
Sorry, I just added this line for debug.
I delited it.